frontend/appflowy_flutter/lib/workspace/presentation/widgets/edit_panel/panel_animation.dart

```diff
@@ -30,12 +30,12 @@ class AnimatedPanelState extends State<AnimatedPanel> {
     final Offset closePos = Offset(widget.closedX, widget.closedY);
     final double duration = _isHidden && widget.isClosed ? 0 : widget.duration;
     return TweenAnimationBuilder(
-      curve: widget.curve ?? Curves.easeOut,
+      curve: widget.curve ?? Curves.easeInOut,
       tween: Tween<Offset>(
         begin: !widget.isClosed ? Offset.zero : closePos,
         end: !widget.isClosed ? Offset.zero : closePos,
       ),
-      duration: Duration(milliseconds: (duration * 1000).round()),
+      duration: Duration(milliseconds: (duration * 800).round()),
```

Comment:
Please revert or change to Duration(seconds: duration). This multiply by 1000 is supposed to convert from seconds to milliseconds.

Reply:
Okay got it !!